.circleci/config.yml

```diff
@@ -19,17 +19,83 @@ jobs:
   pip-install-test:
     executor: python/default
     steps:
-        - run:
-              name: "Test pip"
-              command: echo "pytest" >> requirements.txt
-        - python/load-cache
-        - python/install-deps
-        - python/save-cache
-        - run:
-            name: "Test"
-            command: |
-              # pytest would be a dep in requirements.txt
-              pytest --version
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+          app-dir: ~/project/sample
+      - run:
+          name: "Test"
+          workding_directory: ~/project/sample
+          command: |
+            ls -l
+            # pytest would be a dep in requirements.txt
+            pytest
+
+  uninstall-deps-test:
+    executor: python/default
+    steps:
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+          app-dir: ~/project/sample
+          pypi-cache: false
+      - run:
+          name: "Test"
+          command: |
+            pytest --version
+      - python/uninstall-deps:
+          package-names: pytest
+      - run:
+          name: "Test"
+          command: |
+            pytest --version || echo "successfully uninstalled"
+
+  dist-test:
+    executor: python/default
+    steps:
+      - checkout
+      - run:
+          name: Create setup.py to test dist command
+          command: |
+            echo 'import setuptools; setuptools.setup(name="tmp-example",version="0.0.1",author="null",author_email="null",url="null",description="null")' > setup.py
```

Comment:
Should this be commited to the repository?

Reply:
The sample project is a poetry based project, this doesn't need a setup.py. A few lines below, you'll also see I'm creating a Pipfile as well. I suppose I could have a sample_poetry, sample_pipenv, and sample_pip, and copy the tests between them all (they need to be in the package)

Reply:
I think I got confused and assumed it was part of the orb - my bad!